langchain/package.json

```diff
@@ -74,6 +74,7 @@
   "author": "Langchain",
   "license": "MIT",
   "devDependencies": {
+    "@aws-sdk/client-s3": "^3.306.0",
```

Comment:
This should be added as a peer dependency as well

Reply:
Done!

Reply:
@jasondotparse can you add the optional flag under `peerDependenciesMeta` like the others?